src/structures/MessageAttachment.js

```diff
@@ -86,6 +86,14 @@ class MessageAttachment {
       this.size = data.size;
     }
 
+    if ('filename' in data) {
+      /**
+       * The name of this attachment
+       * @type {string}
+       */
+      this.name = data.filename;
```

Comment:
Isn't this out of the scope of the PR? The property is documented but it was never used until now (honestly not sure why)

Reply:
Yeah I was unsure aswell while looking thru the code so I figured why not add it. Anyway I think I'll leave this in for now

Reply:
actually nvm @ImRodry I'm removing this